.travis.yml

```diff
@@ -10,45 +10,28 @@ env:
     - PATH="$PYENV_ROOT/bin:$PATH"
     - PIP_CACHE_DIR="$HOME/.cache/pip"  # unify pip cache location for all platforms
 stages:
-#  - check
-#  - test
+  - test
   - build
-# Travis doesn't support testing python on Mac yet, so we need
-# to workaround it by installing it directly with brew.
-#
-# Note that we specify language as ruby-2.4.3 to help us satisfy
-# fpm dependencies, as travis runs el capitan with an old ruby
-# version, that causes fpm to crash.
-#
 # Each job will use the previous job's `stage` tag if not specified.
 jobs:
   include:
-  # patch formatting checks
-#  - name: "Check patch formatting"
-#    stage: check
-#    os: linux
-#    language: python
-#    python: 3.7
-#    before_install:
-#    install:
-#    script: ./scripts/ci/check_patch.sh
```

Comment:
How about we enable patch style checks too? Black should help keeping the style consistent and flake8 will point out some issues. Maybe even pylint might be easy to enable at this point. Maybe not in this PR, but just wanted to mention this.

Reply:
@MaxRis @shcheklein what do you think, guys?

Reply:
that would super helpful! I would just want to keep the scope of this one reasonable (we already went from service accounts support into implementing Travis) let's make a separate ticket for this?

Reply:
Yeah, sure.